Content.Server/_Starlight/Railroading/RailroadRuleSystem.cs

```diff
@@ -0,0 +1,208 @@
+﻿using System;
+using System.Linq;
+using Content.Server.GameTicking.Rules;
+using Content.Server.Mind;
+using Content.Shared._Starlight.Abstract.Conditions;
+using Content.Shared._Starlight.Railroading;
+using Content.Shared.Alert;
+using Content.Shared.GameTicking.Components;
+using Content.Shared.Starlight;
+using Content.Shared.Store;
+using Robust.Server.Player;
+using Robust.Shared.Map;
+using Robust.Shared.Prototypes;
+using Robust.Shared.Random;
+using static Content.Shared._Starlight.Railroading.RailroadRuleComponent;
+
+namespace Content.Server._Starlight.Railroading;
+
+public sealed partial class RailroadRuleSystem : GameRuleSystem<RailroadRuleComponent>
+{
+    // it’s something that should be synchronized across all rules.
+    private const byte CardPerUser = 3;
```

Comment:
*maybe* should have this defined in the railroadable component itself, but this is also fine for now